File: library/src/main/java/com/getstream/sdk/chat/notifications/StreamFirebaseMessagingService.java
package com.getstream.sdk.chat.notifications;

import androidx.annotation.NonNull;

import com.getstream.sdk.chat.StreamChat;
import com.google.firebase.messaging.FirebaseMessagingService;
import com.google.firebase.messaging.RemoteMessage;

public class StreamFirebaseMessagingService extends FirebaseMessagingService {

    @Override
    public void onMessageReceived(@NonNull RemoteMessage remoteMessage) {
        StreamChat.getNotificationsManager().onReceiveFirebaseMessage(remoteMessage, this);
    }

    @Override
    public void onNewToken(@NonNull String token) {
        StreamChat.getNotificationsManager().setFirebaseToken(token, this);
    }
}

File: library/src/main/java/io/getstream/chat/android/client/notifications/StreamFirebaseMessagingService.kt
package io.getstream.chat.android.core.poc.library.notifications

import com.google.firebase.messaging.FirebaseMessagingService
import com.google.firebase.messaging.RemoteMessage

class StreamFirebaseMessagingService: FirebaseMessagingService() {

    override fun onMessageReceived(remoteMessage: RemoteMessage) {
        //StreamChat.getNotificationsManager().onReceiveFirebaseMessage(remoteMessage, this)
    }

    override fun onNewToken(token: String) {
        //StreamChat.getNotificationsManager().setFirebaseToken(token, this)
    }
}